pkg/app/pagewriter/pagewriter_test.go

```diff
@@ -19,6 +19,10 @@ var _ = Describe("Writer", func() {
 		var writer Writer
 		var opts Opts
 		var request *http.Request
+		providerName := make([]string, 0)
+		providerName = append(providerName, "<ProviderName>")
+		providerIDArray := make([]string, 0)
+		providerIDArray = append(providerIDArray, "0")
```

Comment:
We can condense this
```suggestion
		providerName := []string{"<ProviderName>"}
		providerIDArray := []string{"0"}
```